DFE-Digital/early-careers-framework
# CPDEL 831 update nqt plus 1 form pages

## Ticket and context

Ticket: [831](https://dfedigital.atlassian.net/jira/software/projects/CPDEL/boards/89?assignee=5f3543bb3e9e2e004db3189d&selectedIssue=CPDEL-831)
Also [768](https://dfedigital.atlassian.net/jira/software/projects/CPDEL/boards/89?assignee=5f3543bb3e9e2e004db3189d&selectedIssue=CPDEL-768) - The error for restricting a user that has already added teachers **has not** been included in this PR. 

## Tech review

### Is there anything that the code reviewer should know?

The prototype is [here](https://ecf-engage-and-learn-prototype.herokuapp.com/nqt1/it3/choose).

### Code quality checks

### HTML Checks

### Gotchas

## Product review

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Login as steven.woods@digital.education.gov.uk - get the link to access the service from notify. 
3. Go to `https://ecf-review-pr-978.london.cloudapps.digital/schools/000004-zz-test-school-4/cohorts/2021`
4. Have a click around, it should also send the email and show in notify once you've confirmed the users to be added. 

**note** - users will not need to be logged in to add teachers as they will access the page from an invite to add nqt plus ones. This step is not included in this checking so the next best thing is to be logged in.
